circus/util.py

```diff
@@ -693,11 +693,14 @@ def configure_logger(logger, level='INFO', output="-", loggerconfig=None,
         elif output.startswith('syslog://'):
             # URLs are syslog://host[:port]?facility or syslog:///path?facility
             info = urlparse(output)
-            facility = 'user'
```

Comment:
This is not related to the original issue. The default facility can be kept as is here. Please revert this part along with the `else` clause later on.